grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandler.java

```diff
@@ -44,12 +47,22 @@
      * Returns a plain text or json response based on the content type.
      */
     static UnframedGrpcErrorHandler of() {
+        return of(UnframedGrpcStatusMappingFunction.of());
+    }
+
+    /**
+     * Returns a plain text or json response based on the content type.
+     *
+     * @param statusMappingFunction The function which maps the {@link Throwable} or gRPC {@link Status} code
+     *                              to an {@link HttpStatus} code.
+     */
+    static UnframedGrpcErrorHandler of(UnframedGrpcStatusMappingFunction statusMappingFunction) {
```

Comment:
After applying https://github.com/line/armeria/pull/3948#discussion_r770203787, need to `orElse` before returning a lambda expression.
```java
statusMappingFunction = requireNonNull(statusMappingFunction, "statusMappingFunction").orElse(UnframedGrpcStatusMappingFunction.of());
```